File: apps/vscode/test/integration/fixture/AgentContextFixture.ts
class AgentContextFixture {
  /* @agent-context invariant
   * This annotation must remain independently filterable from the method and class folds.
   */
  run(): number {
    if (Date.now() > 0) {
      return 1;
    }
    return 0;
  }

  /* @agent-context history
   * Keep this second annotation independent so selective filtering is observable.
   */
  previousResult(): number {
    return 1;
  }
}

export { AgentContextFixture };

File: apps/vscode/test/integration/fixture/AgentContextFixture.java
final class AgentContextFixture {
    /* @agent-context invariant
     * Native Java comment, method, and class folds must coexist with HumanEye.
     */
    int run() {
        if (System.nanoTime() > 0) {
            return 1;
        }
        return 0;
    }
}
